magento/pwa-studio
# fix: add to list button should be centered

## Description


## Related Issue







Closes #3350.

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature





1. Verify heart icon/button is aligned with "add to cart button" on desktop.
2. Verify it on mobile.

#### Test scenario(s) for any existing impacted features/areas





1. Verify heart icon is still aligned properly on other pages that use it (I think just the product page at this time. Maybe cart?)

#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
